script.skin.helper.colorpicker/LICENSE

```diff
@@ -0,0 +1,340 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 2, June 1991
```

Comment:
You are bundling 2 license files. Why not keep only the .txt file?

Reply:
Was there by Marcel. I'll remove it. I just took care for the compatiblity.